src/ripple/net/impl/RPCCall.cpp

```diff
@@ -497,14 +497,26 @@ class RPCParser
 
     bool isValidJson2(Json::Value const& jv)
     {
+        if (jv.isArray())
+        {
+            if (jv.size() == 0)
+                return false;
+            for (Json::UInt i = 0; i < jv.size(); ++i)
```

Comment:
I don't think it will make much difference either way.  I'll code it up with range-for and then pick which ever one I like the best.  Good suggestion, thanks.